using-timescaledb/continuous-aggregates.md

```diff
@@ -121,116 +118,111 @@ so that it contains the aggregates computed across the entire
 continuous aggregates since they are not possible to parallelize by
-PostgreSQL. In addition, TimescaleDB continuous aggregates does not
+PostgreSQL. In addition, TimescaleDB continuous aggregates do not
 currently support the `FILTER` clause (not to be confused with
 `WHERE`) even though it is possible to parallelize but we might add
 support for this in a future version.
 
 #### Automatic refresh with a continuous aggregate policy
 
-You can refresh the continuous aggregate view manually as mentioned
-above, but you can also automate the refresh by adding a continuous
-aggregate policies to automatically refresh data.
+Continuous aggregate policies can be configured to support different
+use cases. For example, you might want to:
 
-There are a few situations that you might want to automate:
+- have the continuous aggregate and the hypertable be in sync, even
+  when data is removed from the hypertable, or
+- keep the aggregate data in the continuous aggregate when removing
+  source data from the hypertable.
 
-- You want the continuous aggregate and the hypertable to be in sync,
-  even when data is removed from the hypertable.
-- You want to aggregate data from the hypertable into the continuous
-  aggregate but want to keep the aggregated data in the continuous
-  aggregate when removing data from the hypertable.
-
-These cases can be automatied using a *continuous aggregate policy*,
-which are added using the function
+These use cases are supported by different configuration to
 [`add_continuous_aggregate_policy`][add-continuous-aggregate-policy].
 
-This function takes takes three arguments:
+This function takes three arguments:
 
-- The parameter `start_interval` indicate the start of the refresh
+- The parameter `start_offset` indicates the start of the refresh
   window relative to the current time when the policy executes.
-- The parameter `end_interval` indicate the end of the refresh window
+- The parameter `end_offset` indicates the end of the refresh window
   relative to the current time when the policy executes.
-- The parameter `schedule_interval` indicate the refresh interval in
+- The parameter `schedule_interval` indicates the refresh interval in
   wall-clock time.
 
 Similar to the `refresh_continuous_aggregate` function, providing
-`NULL` to `start_interval` or `end_interval` makes the range
-open-ended and will extend to the beginning or end of time,
-respectively.
+`NULL` to `start_offset` or `end_offset` makes the range open-ended
+and will extend to the beginning or end of time,
+respectively. However, it seldom makes sense to use `NULL` for the
+`end_offset`. Instead, it is recommended to set the `end_offset` such
+that at least one time bucket is excluded. For time-series data that
+see mostly in-order writes, the time buckets that still see lots of
+writes will quickly have out-of-date aggregates. Eliding the buckets
+will also provide better performance.
```

Comment:
Slight rewrite. I do not think you need that last sentence though
```suggestion
and will extend to the beginning or end of time,
respectively. However, it seldom makes sense to use `NULL` for the
`end_offset`. Instead, it is recommended to set the `end_offset` so
that at least the last time bucket is excluded. Since time-series data usually
see mostly writes to the end of the table, any aggregate on this bucket will quickly become out-of-date. Eliding the last buckets
will also provide better performance.
```

Reply:
FWIW, I wanted to avoid the use of "last bucket" because it is a bit ambiguous and there is really no choice as to what buckets `end_offset` will exclude. I did some tweaks to the text, however.

Reply:
Using "newest buckets" then? The text just mention "buckets", which makes it confusing to what bucket is actually referenced.